docs/intent_firing.md

```diff
@@ -2,7 +2,24 @@
 title: "Firing intents"
 ---
 
-When you fire an intent, you will get a response back or an error will be raised. It is up to the component to return the result to the user.
+If your code matches user's speech or text to intents, you can let the intent be handled by Home Assistant. This can be done from inside your own integration, or via the generic Intent handle API.
```

Comment:
**the** user's

Reply:
```suggestion
If your code matches the user's speech or text to intents, you can let the intent be handled by Home Assistant. This can be done from inside your own integration, or via the generic Intent handle API.
```